SmartLink.Host/app.py

```
@@ -0,0 +1,62 @@
+from flask import Flask, jsonify, request
```

Ruff (0.8.2)

`flask.request` imported but unused

Remove unused import: `flask.request`

(F401)